os/arch/arm/src/amebad/amebad_serial.c

```diff
@@ -195,6 +196,7 @@ struct rtl8721d_up_dev_s {
 #ifdef CONFIG_SERIAL_OFLOWCONTROL
 	uint8_t oflow:1;			/* output flow control (CTS) enabled */
 #endif
+	uint8_t tx_level;
```

Comment:
Since this tx_level has been added only for data loss while flash write , do you think we should add appropriate CONFIG dependency for this ?

Reply:
Actually, this change is not limited to one application. From the test results, it can generally reduce the total number of uart tx interrupts so that reduce the time used for the transmission.